lib/mongo/bulk_write/result.rb

```diff
@@ -96,8 +101,11 @@ def deleted_count
       # @param [ BSON::Document, Hash ] results The results document.
       #
       # @since 2.1.0
-      def initialize(results)
+      #
+      # @api private
+      def initialize(results, acknowledged)
```

Comment:
`acknowledged` needs to be documented in the docstring.

Reply:
done